pallets/manta-pay/src/benchmark/mod.rs

```diff
@@ -44,30 +44,30 @@ where
 
 /// Init assets for manta-pay
 #[inline]
-pub fn init_asset<T>(owner: &T::AccountId, id: AssetId, value: Balance)
+pub fn init_asset<T>(owner: &T::AccountId, id: u128, value: Balance)
```

Comment:
i thought mantapay does not yet change to `u128` in this PR?
Why not use `StandardAssetId` here?

Reply:
Nice catch, will change it. This code is a copy paste from the manta-pay-v1 branch to get it to compile.